engines/joomla/nucleus/particles/contentarray.html.twig

```diff
@@ -97,12 +97,14 @@
                                             {% set category_link = display.category.enabled == 'link' %}
                                             <span class="g-array-item-category">
                                                 {% for category in article.categories %}
-                                                    {% if category_link %}
-                                                        <a href="{{ category.route }}">
+                                                    {% if loop.last %}
```

Comment:
Hey Djamil,

The first thing I tried was actually `{% for category in article.categories|last %}` but it doesn't work, no category is shown at all.

I'm not sure I understand your second comment. Could you please paste the code as you want it to be?

The only way I could make it work was the one I committed 😉